src/transformers/modeling_tf_distilbert.py

```diff
@@ -234,16 +232,17 @@ def unshape(x):
         k = shape(self.k_lin(key))  # (bs, n_heads, k_length, dim_per_head)
         v = shape(self.v_lin(value))  # (bs, n_heads, k_length, dim_per_head)
 
-        q = q / math.sqrt(dim_per_head)  # (bs, n_heads, q_length, dim_per_head)
+        q = tf.cast(q, dtype=tf.float64)
+        q = tf.divide(q, tf.math.sqrt(tf.cast(dim_per_head, dtype=tf.float64)))  # (bs, n_heads, qlen, dim_per_head)
```

Comment:
Would a reciprocal square root be more intuitive here? Multiplications tends to be faster than division:

`tf.multiply(q, tf.math.rqsrt(...))`